virt build fails if the lbz2 development package are installed

As reported through issue #40, the virt build fails when the lbz2 packages are installed:

```
 GEN     trace/generated-helpers.c
  CC      x86_64_virt-softmmu/trace/control-target.o
  CC      x86_64_virt-softmmu/gdbstub-xml.o
  CC      x86_64_virt-softmmu/trace/generated-helpers.o
  LINK    x86_64_virt-softmmu/qemu-system-x86_64_virt
../block/dmg-bz2.o: In function `dmg_bz2_init':
/home/wkozaczuk/projects/nemu/block/dmg-bz2.c:59: undefined reference to `dmg_uncompress_bz2'
/home/wkozaczuk/projects/nemu/block/dmg-bz2.c:60: undefined reference to `dmg_uncompress_bz2'
collect2: error: ld returned 1 exit status
Makefile:199: recipe for target 'qemu-system-x86_64_virt' failed
make[1]: *** [qemu-system-x86_64_virt] Error 1
Makefile:481: recipe for target 'subdir-x86_64_virt-softmmu' failed
make: *** [subdir-x86_64_virt-softmmu] Error 2
```

That's because we disable dmg, but since the lbz2 devel package is installed, we end up with a config-host.mak with `CONFIG_BZIP2=y`.

--- a/tools/build_aarch64.sh
+++ b/tools/build_aarch64.sh
@@ -15,6 +15,7 @@ $SRCDIR/configure \
  --disable-libssh2 \
  --disable-linux-aio \
  --disable-lzo \
+ --disable-bzip2 \
  --disable-modules \
  --disable-netmap \
  --disable-qom-cast-debug \

--- a/tools/build_x86_64.sh
+++ b/tools/build_x86_64.sh
@@ -15,6 +15,7 @@ $SRCDIR/configure \
  --disable-libssh2 \
  --disable-linux-aio \
  --disable-lzo \
+ --disable-bzip2 \
  --disable-modules \
  --disable-netmap \
  --disable-qom-cast-debug \

--- a/tools/build_x86_64_virt.sh
+++ b/tools/build_x86_64_virt.sh
@@ -15,6 +15,7 @@ $SRCDIR/configure \
  --disable-libssh2 \
  --disable-linux-aio \
  --disable-lzo \
+ --disable-bzip2 \
  --disable-modules \
  --disable-netmap \
  --disable-qom-cast-debug \

